release/rllib_tests/8gpus_96cpus_gce.yaml

```diff
@@ -3,13 +3,18 @@ region: us-west1
 allowed_azs:
     - us-west1-b
 
-max_workers: 0
+max_workers: 1
 
 head_node_type:
     name: head_node
-    instance_type: n1-highmem-96-nvidia-tesla-v100-8 # g4dn.metal
+    instance_type: n1-standard-64-nvidia-tesla-t4-4
```

Comment:
this should be n1-standard-32 i think

Reply:
Actually, since some of these tests require a lot of CPUs, this is a 96 CPU cluster. So the headnode has 64 CPUs and the worker has 32 CPUs to reach this.